packages/codemods/src/emotion-to-compiled/__tests__/emotion-to-compiled.test.tsx

```diff
@@ -3,7 +3,7 @@ jest.disableAutomock();
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest;
 
-import transformer from '../emotion-to-compiled';
+import { transformer } from '../emotion-to-compiled';
```

Comment:
I'm not a huge fan of exporting the `transformer` just for testing. Is there anything wrong with using the public API through the default import here? It seems like you just need to rename `pluginModule` to `plugin`, and handle passing in the `pluginModule` internally as well as paths.

Reply:
Yeah, this is a mild workaround as testUtils in jscodeshift does not handle async transformations, which breaks testability through those helper libraries